nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/statusbar/jquery.statusbar.js

```diff
@@ -22,7 +22,19 @@
  *
  *
  */
-(function ($) {
+(function (root, factory) {
+    if (typeof define === 'function' && define.amd) {
+        define(['jquery', 'nf.Common'],
+            function ($, nfCommon) {
+                return (nf.CanvasUtils = factory($, nfCommon));
+            });
+    } else if (typeof exports === 'object' && typeof module === 'object') {
+        module.exports = (nf.CanvasUtils = factory(require('d3'), require('nf.Common')));
+    } else {
+        nf.CanvasUtils = factory(root.$, root.nf.Common);
+    }
+}(this, function ($, nfCommon) {
```

Comment:
I suspect the references to `nf.CanvasUtils` here are a copy/paste issue. Generally, our `jquery` plugins do not reference any `nf` classes. The intended direction is for our `nf` classes to reference and use the `jquery` plugins. Is it possible here to instead of calling the formatting code here to instead let the client of the `statusbar` format the bulletins and pass in a `jquery` reference (`$`)?

Reply:
Your observation is correct. It was a copy/paste issue. I fixed this by passing in formatted bulletins and reverted the jquery reference to its original method.